### ISSUE TITLE
Opentoonz "quits unexpectedly" whenever I do anything complicated. 

### ISSUE THREAD

**Issue by TruckerJoe** [opentoonz/opentoonz#326]
I'm super excited to start using Opentoonz, but the application quits whenever I try to do anything complex like render an animation. I have a MacBook Pro 10.11.5 so I believe I match the necessary specs. Does anybody have a solution to this problem? I would really appreciate the help.

Thanks!

**Comment by kogaki** [comment]
We can't help without more information. (e.g. #213 )

* Which version? Can you try latest nightly build?
* What error message says ?

**Comment by kogaki** [comment]
Or, user forum can be your help <https://groups.google.com/forum/#!forum/opentoonz_en>

**Comment by Scheherazade** [comment]
I've been having similar problems to the original poster - I'm on a MacBook Air, using OSX El Capitan. Some of the bugs have been quite hard to consistently reproduce, but all end in the program crashing.

The one problem causing the most trouble for me seems to be with the Control Point Editor when working with vectors. It's a bit of an odd bug, since when I open OpenToonz for the first time, I can edit control points and bezier curve handles just fine for as long as I want. But if I close OpenToonz and then open it up a second time, I can only use the Control Point Editor a limited number of times before OpenToonz "quits unexpectedly". This then becomes standard behaviour each time I open up OpenToonz. This bug was in version 1.0.2, and has been in all the nightly builds since. I've done a clean install every time.

I can provide the error data if you'd like.

**Comment by TruckerJoe** [comment]
Hi Kogaki,

I am using version 1.0.2 and I have used the latest nightly build (just today). The error message says simply: OpenToonz quit unexpectedly. It gives me an option to send a report to apple and to "show details". Unfortunately I can't make any sense of the pages of code that it shows me when I press show details. I'd be happy to include them if you think they could help. 

The problem occurs whenever I attempt to render an animation or after I have spent a while using the software. 

Would love to resolve this problem, thank you for your help.
